test/martha/martha_v3.test.js

```diff
@@ -84,20 +115,22 @@ const fullExpectedResult = (expectedGoogleServiceAccount) => {
 };
 
 const drsObjectWithMissingFields = {
-    id: 'v1_abc-123',
-    description: '123 BAM file',
-    name: '123.mapped.abc.bam',
-    created_time: '2020-04-27T15:56:09.696Z',
-    version: '0',
-    mime_type: 'application/octet-stream',
-    size: 123456
+    "data_object": {
+        id: 'v1_abc-123',
+        description: '123 BAM file',
+        name: '123.mapped.abc.bam',
+        created: '2020-04-27T15:56:09.696Z',
+        version: '0',
+        mime_type: 'application/octet-stream',
+        size: 123456
+    }
 };
```

Comment:
Just noticed this-- that's a DOS response but the name of the variable is still `drsObject...` Which is this block supposed to be: `dos` or `drs`?

It may be as simple as renaming this to `dataObjectServiceObjectWithMissingFields`?

Reply:
renamed to `dosObjectWithMissingFields`